api/models/user.js

```diff
@@ -9,29 +15,34 @@ const associate = ({ User, Build, Site, UserAction }) => {
   });
   User.hasMany(UserAction, {
     foreignKey: 'userId',
+    as: 'userActions',
+  });
+  User.belongsToMany(User, {
+    through: 'user_action',
+    as: 'actionTarget',
+    foreignKey: 'targetId',
+    unique: false,
   });
 };
 
 function toJSON() {
-  const object = this.get({
+  const record = this.get({
     plain: true,
   });
+  /* eslint-disable no-param-reassign */
+  return Object.assign({}, Object.keys(record).reduce((out, attr) => {
```

Comment:
This is cool. We should abstract to a util function and re-use in future code/updates.

Reply:
oooooooh